hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCommitArchiveLog.java

```diff
@@ -143,23 +148,34 @@ public boolean archiveIfRequired() {
     //TODO (na) : Add a way to return actions associated with a timeline and then merge/unify
     // with logic above to avoid Stream.concats
     HoodieTimeline commitTimeline = table.getCompletedCommitTimeline();
+    Optional<HoodieInstant> oldestPendingCompactionInstant =
```

Comment:
can we use "earliest" "latest" to stay consistent with rest of the code?

Reply:
Fixed